wled00/json.cpp

```diff
@@ -872,6 +880,35 @@ void serializePalettes(JsonObject root, AsyncWebServerRequest* request)
   }
 }
 
+void serializeNetworks(JsonObject root)
+{
+  JsonArray networks = root.createNestedArray("networks");
```

Comment:
Use `F("string")` to store strings in flash memory instead or precious RAM.

Reply:
I always forget coming from esp32 development were `F()`  is basically `#define F(x) x` and all constant strings are in flash anyway. I need to remember esp8266 support ;).

Reply:
F() is flash string helper, which is a class method. On the other hand PSTR() is just a moniker on ESP32 that does nothing.